config/envoyconfig/luascripts/drop-check-route-requests.lua

```diff
@@ -0,0 +1,7 @@
+function envoy_on_request(request_handle)
+  local headers = request_handle:headers()
+  if headers:get("x-pomerium-check-route") ~= nil or headers:get(":method") == "TRACE" then
```

Comment:
Does `TRACE` ever occur in the wild?

Reply:
unlikely; it's usually disabled on web servers, and entirely unsupported by browsers. I originally used it here as an extra precaution to basically guarantee that even if somehow both our ext_authz filter and this filter let the request through, it would still be a no-op. However, it is probably not necessary, if that situation happens then we have much bigger problems...